packages/core/core/src/TransformerRunner.js

```diff
@@ -97,6 +97,7 @@ export default class TransformerRunner {
     let {assets, initialAssets} = await this.runPipeline(
       input,
       pipeline,
+      pipeline,
```

Comment:
may be time to convert `runPipeline` to a params object given there are four args and the second and third are both `pipeline` here 😄

Reply:
haha yeah